src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+
+}
+
+.form {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  position: relative;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form--small {
+  width: 100%;
+  height: 42px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:active {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.form__label {
+  display: flex;
+  margin-left: 25px;
+}
+
+.form__label--small {
+  margin-left: 12px;
+}
+
+.text__input {
+  background: transparent;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 800;
+  color: #3d4e61;
+  border: 0;
+  width: 95vw;
```

Comment:
Not the best idea - now the input field is bigger. Try not to pick the closest value. It can be done without hardcoding the right width. Also, I suggest you make the height of the input equal to the height of the form too.

<img width="1345" alt="Screenshot 2023-02-06 at 23 49 57" src="https://user-images.githubusercontent.com/54940595/217096065-2b733586-0a1b-4872-b7d3-84608e8f8a47.png">